cpp/core.cc

```diff
@@ -254,6 +254,27 @@ bool InputAvailable(int fd) {
   return select(FD_SETSIZE, &fds, NULL, NULL, &timeout) > 0;
 }
 
+List<int>* WaitForInputs(List<int>* fds_in) {
+  auto* ret = NewList<int>();
+  fd_set fds;
+  FD_ZERO(&fds);
+  int n = len(fds_in);
+  for (int i = 0; i < n; ++i) {
+    FD_SET(fds_in->at(i), &fds);
+  }
+  int out = select(FD_SETSIZE, &fds, NULL, NULL, NULL);
```

Comment:
yeah here we don't pass the list of "exc"?   So we probably shouldn't pass it in Python, if there are no errors to handle

I'm not sure if that affect "close" or not ...